verification/verify_timestamp.py

```
@@ -0,0 +1,31 @@
+from playwright.sync_api import sync_playwright, expect
+import time
+
+def run():
+    with sync_playwright() as p:
+        browser = p.chromium.launch(headless=True)
+        # Taller viewport to see sidebar content
+        page = browser.new_page(viewport={'width': 1280, 'height': 1200})
+
+        try:
+            page.goto("http://localhost:3000/?mock_user=true", timeout=10000)
+        except:
```

Ruff (0.14.14)

[error] Do not use bare `except`

(E722)